xray/commands/curation/audit.go

```diff
@@ -463,6 +471,14 @@ func (nc *treeAnalyzer) extractPoliciesFromMsg(respError *ErrorsResp) []Policy {
 			cond := polCond[1]
 			policies = append(policies, Policy{Policy: strings.TrimSpace(pol), Condition: strings.TrimSpace(cond)})
 		}
+		if len(polCond) == 4 {
+			pol := polCond[0]
+			cond := polCond[1]
```

Comment:
Is it possible to have 0, 1, or 3 columns? If not, let's put the pol and col outside of the if's, to remove code duplication.
```suggestion
	        pol := polCond[0]
		cond := polCond[1]
		if len(polCond) == 4 {
```